hatasmota/const.py

```diff
@@ -161,6 +167,8 @@
 SENSOR_STATUS_SSID: Final = "status_ssid"
 SENSOR_STATUS_VERSION: Final = "status_version"
 
+SENSOR_STATUS_DEEPSLEEP: Final = "status_deepdleep"
```

Comment:
Let's add the battery one too. The `status_deepsleep` can be held off to the next PR though.
```suggestion
SENSOR_STATUS_BATTERY_PERCENTAGE: Final = "status_battery_percentage"
```